src/CommunityToolkit.Aspire.Hosting.SurrealDb/SurrealDbContainerImageTags.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting;
+
+internal sealed class SurrealDbContainerImageTags
+{
+    public const string Registry = "docker.io";
+    public const string Image = "surrealdb/surrealdb";
+    public const string Tag = "v2.0.4";
```

Comment:
```suggestion
    /// <summary>docker.io</summary>
    public const string Registry = "docker.io";
    /// <summary>surrealdb/surrealdb</summary>
    public const string Image = "surrealdb/surrealdb";
    /// <summary>v2.0</summary>
    public const string Tag = "v2.0";
```
Add triple slash docs because of `inheritdoc` in `example` of  `AddSurrealServer`.

Also, update the tag to `v2.0` because this is our convention for images that support `major.minor` tags.